index.js

```diff
@@ -1,50 +1,89 @@
+require("dotenv").config();
 const axios = require("axios").default;
-const notifier = require('node-notifier');
-const utils = require('./src/utils');
+const notifier = require("node-notifier");
+const path = require('path');
+const utils = require("./src/utils");
+const NodeCache = require("node-cache");
+const myCache = new NodeCache();
 
-const HEB_URL = 'https://heb-ecom-covid-vaccine.hebdigital-prd.com/vaccine_locations.json';
-const SECOND_INTERVAL = .3;
+const HEB_URL = process.env.HEB_API;
+const twilioAccount = process.env.TWILIO_ACCOUNT;
+const twilioAuthToken = process.env.TWILIO_AUTH_TOKEN;
+const client = require("twilio")(twilioAccount, twilioAuthToken);
+
+const SECOND_INTERVAL = 0.3;
 const HTTP_OK = 200;
 
 function notify(arrayOfLocations) {
-  const available = arrayOfLocations.reduce((acc, store)=> `${acc}\nWe found ${store.openAppointmentSlots} slots available at '${store.name}'!`,'');
-  const message = {
-    title: 'COVID Vaccine available!!! ',
-    message: available,
-    icon: ' 🧬 💉', // Absolute path (doesn't work on balloons)
-    sound: true, // Only Notification Center or Windows Toasters
-  };
-  console.log(`🧬 💉 : ${available}`);
-
-  notifier.notify(message);
+  const valores = arrayOfLocations.filter((location) => {
+    const value = myCache.get(location.storeNumber);
+    if (value !== undefined) {
```

Comment:
```suggestion
    if (value !== undefined && value.spots !== location.openAppointmentSlots) {
```

Simplify the if statement